docs/docsite/rst/rhv/faq.rst

```diff
@@ -0,0 +1,34 @@
+.. _RHV_faq:
+
+******************
+Ansible RHV FAQ
+******************
+
+ovirt_* modules
+===============
+
+What is oVirt? What is RHV?
+------------------------------------------------------------
+
+oVirt is the upsteam open-source edition of Red Hat Virtualization. 
+Red Hat Virtualization is an open-source alternative hypervisor to such like VMware. As of RHV version 4.1+ the feature-set and 
+maturity of RHV insists that it is prime for enterprise production workloads, among others.
+
+What version of oVirt / RHV will Ansible modules work with?
+------------------------------------------------------------
+
+Most of the more recently build ``ovirt_*`` modules are best experienced with oVirt/ RHV 4.1 or newer, hence the preview flags 
+on the modules, indicating no guarantee for backwards compatibility.
```

Comment:
Note that we make the modules backward compatible, even when docs says it don't have to be.

Reply:
how far back are they tested to work?

Reply:
It should work on all 4.x versions. The only limitation is SDK, which must be always the latest.